spec/domain/person/filter/list_spec.rb

```diff
@@ -8,44 +8,75 @@
 require 'spec_helper'
```

Comment:
filter passen wir in einem eigenen Ticket an. 
diesen Spec auf einem anderen Feature Branch parkieren?